plexpy/notification_handler.py

```diff
@@ -516,11 +548,26 @@ def build_notify_text(session=None, timeline=None, state=None):
                         'summary': metadata['summary'],
                         'tagline': metadata['tagline'],
                         'rating': metadata['rating'],
-                        'duration': duration
+                        'duration': duration,
+                        'stream_duration': stream_duration,
+                        'remaining_duration': duration - view_offset,
+                        'progress': view_offset,
+                        'progress_percent': progress_percent,
```

Comment:
Duration, stream duration, remaining duration, progress, and progress percent should be there already a few lines above.